cloudbuild_cosign.sh

```diff
@@ -9,15 +9,19 @@ export KMS_VAL=gcpkms://projects/$PROJECT_ID/locations/global/keyRings/cosign/cr
 cosign version
 
 # Sign images with cosign
-for distro_suffix in "" -debian10; do
+for distro_suffix in "" -debian10 -debian11; do
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:nonroot
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:latest
+  cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:debug-nonroot
+  cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:debug
```

Comment:
@dlorenc @dekkagaijin was there a reason these static images weren't previously signed?

Reply:
@priyawadhwa might know, but I don't think so.

Reply:
no reason! i probably forgot to include them 😓